dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -675,20 +675,23 @@ void ExchangeReceiverBase<RPCContext>::readLoop(const Request & req)
     });
 
     CPUAffinityManager::getInstance().bindSelfQueryThread();
+    Stopwatch watch;
     bool meet_error = false;
     String local_err_msg;
     String req_info = fmt::format("tunnel{}+{}", req.send_task_id, req.recv_task_id);
+    ReceiverMode recv_mode = req.is_local ? ReceiverMode::Local : ReceiverMode::Sync;
+    UInt64 waiting_task_time = 0;
```

Comment:
Maybe `sending_data_time` is better?

Reply:
It is the time that `findTunnelWithTimeout` takes, so I think `waiting_task_time` is better here.